convert2rhel/subscription.py

```diff
@@ -250,6 +250,7 @@ def attach_subscription():
     loggerinst = logging.getLogger(__name__)
 
     if tool_opts.activation_key:
+        loggerinst.info("Using Activation Key informed by the user...")
```

Comment:
We should be direct in the messages, i.e. you instead of user. But in this case I'd go with:
```suggestion
        loggerinst.info("Using the activation key provided through the command line...")
```